app/Services/Student/Attendance/IndexService.php

```diff
@@ -25,6 +25,7 @@ public function __invoke(
             'course.chapters.lessons',
             'lessonAttendances',
             'course.tags',
+            'course', 'course.courseDeadline',
```

Comment:
次の記述だけで`course`もEager Loadingの対象になるので、修正をお願いします！
```suggestion
            'course.courseDeadline',
```